DQM/L1TMonitor/interface/L1TObjectsTiming.h

```diff
@@ -103,6 +103,11 @@ class L1TObjectsTiming : public DQMEDAnalyzer {
   std::vector<MonitorElement*> etsum_eta_phi_METHF_lastbunch;
   std::vector<MonitorElement*> etsum_eta_phi_MHT_lastbunch;
   std::vector<MonitorElement*> etsum_eta_phi_MHTHF_lastbunch;
+ 
+  MonitorElement* denominator;
```

Comment:
These denominators are only used for the muons. There are denominators missing for jets, egamma, taus, and etsums.